pymongo/mongo_client.py

```diff
@@ -137,6 +137,10 @@ def __init__(
     ) -> None:
         """Client for a MongoDB instance, a replica set, or a set of mongoses.
 
+        .. warning:: Starting in PyMongo 4.0, ``directConnection`` now has a default value of
+          False instead of None (which is the same as specifying ``directionConnection=True``).
```

Comment:
"(which is the same as specifying ``directionConnection=True``)" is not quite correct. Let's remove that part.

Reply:
Bernie mentioned that fact explicitly in the ticket, so I thought it would be good to include so that user understands that None does not equal False (as I would assume). What about making it more clear by saying that is "functionally" the same? Or is that still not the case.

Reply:
Oh, ok that makes sense. Will leave it removed then.